### PR TITLE
[SYCL] Fix to windows on comment testing

### PR DESCRIPTION
**Pull request by romanovvlad**
Move env section earlier so that $SHA is initialized before run section.
Remove commented out code.

### CODE REVIEW

File: .github/workflows/windows_test_comment_trigger.yml [hunk 1]
```diff
@@ -56,32 +56,23 @@ jobs:
...
-      run: |
-          gh api --method POST -H "Accept: application/vnd.github+json" /repos/$OWNER/$REPO/statuses/$SHA -f state='success' -f target_url='${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}' -f description='The build started2!' -f context='Windows pre-commit'
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         OWNER: ${{ github.repository_owner }}
         REPO: ${{ github.event.repository.name }}
-        SHA: ${{ steps.sha.outputs.result }}
+        SHA: ${{ needs.windows_test_preparation.outputs.PR_SHA }}
+      run: |
+          gh api --method POST -H "Accept: application/vnd.github+json" /repos/$OWNER/$REPO/statuses/$SHA -f state='success' -f target_url='${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}' -f description='The build started2!' -f context='Windows pre-commit'
```
Comment (AlexeySachkov) [inline .github/workflows/windows_test_comment_trigger.yml:69]:
```suggestion
          gh api --method POST -H "Accept: application/vnd.github+json" /repos/$OWNER/$REPO/statuses/$SHA -f state='success' -f target_url='${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}' -f description='The build started!' -f context='Windows pre-commit'
```

File: .github/workflows/windows_test_comment_trigger.yml [hunk 2]
```diff
@@ -56,32 +56,23 @@ jobs:
...
-  #   runs-on: ubuntu-latest
-  #   needs: [windows_test_preparation]
-  #   steps:
-  #   - name: print_ref
-  #     run: |
-  #       echo "REF == = = = = = = ${{github.ref}}"
-  #       echo "REF == = = = = = = ${{ needs.windows_test_preparation.outputs.PR_SHA }}"
+        SHA: ${{ needs.windows_test_preparation.outputs.PR_SHA }}
+      run: |
+          gh api --method POST -H "Accept: application/vnd.github+json" /repos/$OWNER/$REPO/statuses/$SHA -f state='failure' -f target_url='${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}' -f description='The build started2!' -f context='Windows pre-commit'
```
Comment (AlexeySachkov) [inline .github/workflows/windows_test_comment_trigger.yml:78]:
```suggestion
          gh api --method POST -H "Accept: application/vnd.github+json" /repos/$OWNER/$REPO/statuses/$SHA -f state='failure' -f target_url='${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}' -f description='The build started!' -f context='Windows pre-commit'
```

Comment (romanovvlad) [reply .github/workflows/windows_test_comment_trigger.yml:78]: Maybe `Windows testing` as description?

File: .github/workflows/windows_test_comment_trigger.yml (hunk 1 above)
Comment (romanovvlad) [reply .github/workflows/windows_test_comment_trigger.yml]: Updated.

**Review by AlexeySachkov — APPROVED** [review]

**Review by cperkinsintel — APPROVED** [review]
LGTM

File: .github/workflows/windows_test_comment_trigger.yml [hunk 3]
```diff
@@ -7,7 +7,7 @@ on:
 jobs:
   windows_test_preparation:
     runs-on: ubuntu-latest
-    if: ${{ github.event.issue.pull_request && github.event.comment.body == '/start Windows testing' }}
+    if: ${{ github.event.issue.pull_request && (github.event.comment.body == '/start Windows testing' ||  github.event.comment.body == '/windu please')}}
```
Comment (bader) [inline .github/workflows/windows_test_comment_trigger.yml:10]: Is this change intentional?

**Review by bader — COMMENTED** [review]
"[SYCL] Fix to windows on comment testing" -> "[SYCL] Fix windows testing run by the comment"

File: .github/workflows/windows_test_comment_trigger.yml (hunk 3 above)
Comment (romanovvlad) [reply .github/workflows/windows_test_comment_trigger.yml:10]: Yes, do you think this alternative is not acceptable?

Comment (pvchupin) [inline .github/workflows/windows_test_comment_trigger.yml:10]:
```suggestion
    if: ${{ github.event.issue.pull_request && (github.event.comment.body == '/testwin') }}
```
You can keep easter egg if you like, but shorter version will win anyway I think ;)

Comment (bader) [reply .github/workflows/windows_test_comment_trigger.yml:10]: I think it's confusing. Pavel's suggestion below is more reasonable.

Comment (romanovvlad) [reply .github/workflows/windows_test_comment_trigger.yml]: Done.
